src/menu/settings_menu.rs

```diff
@@ -120,6 +121,25 @@ impl Default for BehaviorMenuEntry {
     }
 }
 
+#[derive(Debug, Clone, Copy, Eq, PartialEq)]
+enum LinksMenuEntry {
+    Title,
+    DiscordLink,
+    GithubLink,
+    DocsLink,
+    TributeLink,
+    GeneralLink,
+    ModdingLink,
+    GetplusLink,
+    Back,
+}
+
+impl Default for LinksMenuEntry {
+    fn default() -> Self {
+        LinksMenuEntry::DiscordLink
+    }
+}
+
```

Comment:
This can be simplified to:

```rs
#[derive(Debug, Clone, Eq, PartialEq)]
enum LinksMenuEntry {
    Title,
    Link(&'static str),
    Back,
}

impl Default for LinksMenuEntry {
    fn default() -> Self {
        LinksMenuEntry::Link(DISCORD_LINK)
    }
}
```